documentation/package.json

```diff
@@ -32,6 +32,7 @@
     "npm-to-yarn": "^1.0.1",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
+    "react-live": "maksnester/react-live#mn/disable-transpilation-options-dist",
```

Comment:
Dafuc?

Reply:
😂

Reply:
Ah. It's the branch ofc. Nice!

Reply:
@larsmoa Fair question. You can specify github `repo#branch` instead of the version number. In that case, during installation the package `node_modules/react-live` will be downloaded from [github repo](https://github.com/maksnester/react-live/tree/mn/disable-transpilation-options-dist) instead of NPM. That's a temporary hack until PR will be merged. I didn't want to create a fork of react-live under org and maintain it and publish it under new name. I just want my patch to work. 

Once my referenced PR to react-live will be merged I'll update it here with new patched react-live version.